### ISSUE TITLE
jsxLoader: Uncaught SyntaxError: Unexpected token ','

### ISSUE THREAD

**Issue by nifgraup** [dataformsjs/dataformsjs#21]
```
<html>
  <head>
    <script src="https://cdn.jsdelivr.net/npm/dataformsjs@5.10.4/js/react/jsxLoader.js"></script>
    <script src="https://unpkg.com/react@16.7.0/umd/react.production.min.js"></script>
  </head>
  <body>
    <script data-type="module" type="text/babel">
      console.log(
        <div>
          <div />
          {[].map(() => <div />)}
        </div>
      );
    </script>
  </body>
</html>
```
gives error

```
lovejoy.html:5 Uncaught SyntaxError: Unexpected token ','
    at addToPage (jsxLoader.js:389)
    at jsxLoader.js:396
    at new Promise (<anonymous>)
    at Object.loadScript (jsxLoader.js:393)
    at Object.setup (jsxLoader.js:273)
    at HTMLDocument.<anonymous> (jsxLoader.js:1715)
addToPage @ jsxLoader.js:389
(anonymous) @ jsxLoader.js:396
loadScript @ jsxLoader.js:393
setup @ jsxLoader.js:273
(anonymous) @ jsxLoader.js:1715
```

**Comment by ConradSollitt** [comment]
Hi @nifgraup 

It's been a while but I finally got around to making the update for this issue and the other one that you opened. I gave you credit in the change log for finding and helping create the example code:
https://github.com/dataformsjs/dataformsjs/blob/master/CHANGELOG.md

**Comment by ConradSollitt** [comment]
Fixed with the latest release 5.10.6.
